ui/nuxeo-data-table/data-table-cell.js

```diff
@@ -28,6 +28,16 @@ import './data-table-templatizer-behavior.js';
           :host([hidden]) {
             display: none;
           }
+
+          :host([actions='single']) {
```

Comment:
Here, the only thing that changes from 'single' to 'double' seems to be part of the flex, that doubles in fact.
Would it be interesting to have a default one (e.g. the 'single') and an exception?

Reply:
I agree, please checkout the new solution.